schemas/address-translations-schema.json

```diff
@@ -0,0 +1,293 @@
+                    "description": "The street and premises number of the address"
+                },
+                "building": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The building name or number of the address"
+                },
+                "postOfficeBox": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The post office box information of this address"
+                },
+                "locality": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the city/town/village of this address"
+                },
+                "dependentLocality": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the dependent (sub) locality of this address"
+                },
+                "region": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the state/province/etc. of this address"
+                },
+                "subregion": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the subregion of this address"
+                },
+                "subsubregion": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the subsubregion of this address"
+                },
+                "postalCode": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The postal code/postcode/zip code of the address"
+                },
+                "postOffice": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the post office of the address"
+                },
+                "country": {
+                    "$ref": "#/definitions/fieldMetadata",
+                    "description": "The name of the country of this address"
+                }
+            }
+        },
+        "fieldNames": {
+            "type": "object",
+            "description": "Translations of the various types of fields in a possible address",
+            "properties": {
+                "Area": {
+                    "type": "string",
+                    "description": "Translation of this field type"
+                },
+                "Autonomous Community": {
+                    "type": "string",
+                    "description": "Translation of this field type"
+                },
+                "Band": {
+                    "type": "string",
+                    "description": "Translation of this field type"
+                },
+                "Borough": {
+                    "type": "string",
+                    "description": "Translation of this field type"
+                },
+                "Burgh": {
```

Comment:
these property data is pretty specific. where did you get this information?

Reply:
If you look at the various references at the bottom of specs/Address.md, there is a description of the various types of regions and localities around the world. The "Autonomous Community" and "Band" regions come from the addresses in native American reservations.

Reply:
oh, thank you.